src/validation.cpp

```diff
@@ -1126,7 +1129,72 @@ bool MemPoolAccept::PackageMempoolChecks(const std::vector<CTransactionRef>& txn
         // This is a package-wide error, separate from an individual transaction error.
         return package_state.Invalid(PackageValidationResult::PCKG_POLICY, "package-mempool-limits", util::ErrorString(result).original);
     }
-   return true;
+
+    // No conflicts means we're finished. Further checks are all RBF-only.
+    if (!m_rbf) return true;
+
+    // We're in package RBF context; replacement proposal must be size 2
+    if (workspaces.size() != 2) {
+        return package_state.Invalid(PackageValidationResult::PCKG_POLICY, "package RBF failed: replacing cluster not size two");
```

Comment:
Should all these failures be `PCKG_POLICY`? We have individual errors which are reported as well, and now with 1P1C relay, we probably want to act on failures at p2p layer?

Reply:
update: in master we're now processing these `PCKG_POLICY` cases, so should be ok? @glozow

Reply:
IIUC It will leave the "individual failures" in place which should be:
parent: TX_RECONSIDERABLE
child: TX_MISSING_INPUTS